mobile/bazel/kotlin_test.bzl

```diff
@@ -88,32 +36,49 @@ def envoy_mobile_android_test(name, srcs, native_lib_name, deps = [], native_dep
         exports = deps,
         testonly = True,
     )
-    kt_android_local_test(
-        name = name,
-        srcs = srcs,
-        data = native_deps,
-        deps = deps + [
-            repository + "//bazel:envoy_mobile_test_suite",
-            "@maven//:androidx_annotation_annotation",
-            "@maven//:androidx_test_core",
-            "@maven//:androidx_test_ext_junit",
-            "@maven//:androidx_test_runner",
-            "@maven//:androidx_test_monitor",
-            "@maven//:androidx_test_rules",
-            "@maven//:org_robolectric_robolectric",
-            "@robolectric//bazel:android-all",
-            "@maven//:junit_junit",
-            "@maven//:org_mockito_mockito_inline",
-            "@maven//:org_mockito_mockito_core",
-            "@maven//:com_squareup_okhttp3_okhttp",
-            "@maven//:com_squareup_okhttp3_mockwebserver",
-            "@maven//:com_squareup_okio_okio_jvm",
-            "@maven//:org_hamcrest_hamcrest",
-            "@maven//:com_google_truth_truth",
-        ],
-        manifest = repository + "//bazel:test_manifest.xml",
-        custom_package = "io.envoyproxy.envoymobile.tests",
-        test_class = "io.envoyproxy.envoymobile.bazel.EnvoyMobileTestSuite",
-        jvm_flags = jvm_flags(native_lib_name),
-        exec_properties = exec_properties,
-    )
+    dependencies = deps + [
+        repository + "//bazel:envoy_mobile_test_suite",
+        "@maven//:androidx_annotation_annotation",
+        "@maven//:androidx_test_core",
+        "@maven//:androidx_test_ext_junit",
+        "@maven//:androidx_test_runner",
+        "@maven//:androidx_test_monitor",
+        "@maven//:androidx_test_rules",
+        "@maven//:org_robolectric_robolectric",
+        "@maven//:junit_junit",
+        "@maven//:org_mockito_mockito_inline",
+        "@maven//:org_mockito_mockito_core",
+        "@maven//:com_squareup_okhttp3_okhttp",
+        "@maven//:com_squareup_okhttp3_mockwebserver",
+        "@maven//:com_squareup_okio_okio_jvm",
+        "@maven//:org_hamcrest_hamcrest",
+        "@maven//:com_google_truth_truth",
+        "@maven//:org_robolectric_shadows_framework",
```

Comment:
we're currently not using this library, can you give some background on why we're introducing it?

Reply:
Yes, they are all used. I simply created a variable so that they can be shared between `android_local_test` and `kt_android_local_test`.